README.md

```diff
@@ -1 +1,20 @@
-# syncer
\ No newline at end of file
+# syncer
+
+Syncs galaxy integration forks with upstream
+
+## Example Usage (workflow)
+```yaml
+name: sync-upstream
+on:
+  schedule:
+    - cron: '30 5,17 * * *'
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GOG-Nebula/syncer@v0
+        with:
+          upstream: owner/repo
+          reviewers: username1,username2
+```
+***NOTE: should not be used in an integration (eg. cron value is just a random value)***
```

Comment:
I just meant that cause I took the example off the internet. The example was mainly to show its parameters and how it would be used in a workflow. For example, the cron value I haven't even bothered to check what it means. We could and should update the example to something that closely matches our usage after we test it.